Samples/WindowsCopilotRuntime/cs-wpf-dotnet/runDotNetBuild.ps1

```diff
@@ -70,12 +71,15 @@ if (-not (Test-Path $pfx))
     Set-Content -Path $cert_thumbprint -Value $thumbprint -Force
     
     # Export the certificate
-    $cer = Join-Path $user_path 'wcrforwpf.certificate.test.cer'
+    $cer = Join-Path $user_path 'wcrforwpf.certificate.sample.cer'
     $export_cer = Export-Certificate -Cert $cert -FilePath $cer -Force
     $cert_personal = "cert:\CurrentUser\My\$thumbprint"
-    $pfx = Join-Path $user_path 'wcrforwpf.certificate.test.pfx'
+    $pfx = Join-Path $user_path 'wcrforwpf.certificate.sample.pfx'
     $export_pfx = Export-PfxCertificate -Cert $cert_personal -FilePath $pfx -Password $password    
 }
 
-Write-Host "Running 'dotnet build /p:platform=$Platform'"
-dotnet build /p:platform=$Platform
\ No newline at end of file
+Write-Host "Running 'dotnet build /p:platform=$Platform /p:configuration=$Configuration'"
+dotnet build /p:platform=$Platform /p:configuration=$Configuration
+
+Write-Host "`nPlease Install '$PSScriptRoot\WCRforWPF\.user\wcrforwpf.certificate.sample.cer' to Local Machine/Trusted People Store before running"
+Write-Host "`nYou can find the MSIX at $PSScriptRootWCRforWPF\bin\x64\Release\net9.0-windows10.0.22621.0\win-x64\AppPackages\WCRforWPF_Test"
```

Comment:
>win-x64 [](http://example.com/codeflow?start=105&length=7)

Not 'x64' nor 'win-x64' for me. Mine is "ARM64" and "arm64".

Reply:
OH yea I should use the parameter here